corehq/apps/export/models/new.py

```diff
@@ -168,13 +168,7 @@ def create_default_from_export_item(table_path, item, app_ids_and_versions):
             "is_advanced": is_case_update or False,
         }
 
-        if item.tag == PROPERTY_TAG_ROW:
-            column_class = RowNumberColumn
-            constructor_args["repeat"] = len([node for node in table_path if node.is_repeat])
-        else:
-            column_class = ExportColumn
-
-        column = column_class(**constructor_args)
+        column = ExportColumn(**constructor_args)
```

Comment:
i think this should be `SplitExportColumn` when the item is multiple choice and expand questions is set to true